docs/docs/1.2.3/policies/protocol-support-in-kuma.md

```diff
@@ -70,3 +70,43 @@ Effectively `kuma-dp` will send the raw original TLS request as-is to the final
 
 Note that in this case no advanced HTTP or GRPC statistics or logging are available. As a best practice - since Kuma will already secure the traffic across services via the [mutual TLS](../mutual-tls) policy - we suggest disabling TLS in the original services in order to get L7 metrics and capabilities.
 
+## Websocket support
+
+Kuma out of the box support's `Websocket` protocol. The service exposing `Websocket` should be annotated with `kuma.io/protocol: tcp` annotation
+
+As `Websockets` use pure `TCP` connections under the hood, your service have to be recognised by `Kuma` as the `TCP` one. It's also the default behavior for Kuma to assume the service's `inbound` interfaces are the TCP ones, so you don't have to do anything, but if you want to be explicit, you can annotate your services exposing `Websocket` endpoints with `kuma.io/protocol: tcp` annotation. I.e.:
+
```

Comment:
I don't understand this. websocket is an upgrade over HTTP, so why do we need to specify TCP here? Is this actually a TCP server on the backend or a websocket one?